crates/chia-protocol/src/program.rs

```diff
@@ -169,6 +169,14 @@ use pyo3::types::{PyList, PyTuple};
 #[cfg(feature = "py-bindings")]
 use pyo3::exceptions::*;
 
+#[cfg(feature = "py-bindings")]
+#[allow(clippy::needless_pass_by_value)]
+// We use this for the map function, so using a reference here is not ideal.
+fn map_pyerr(err: EvalErr) -> PyErr {
```

Comment:
```suggestion
fn map_pyerr(err: &EvalErr) -> PyErr {
```

I would think this would also silence clippy, and you could remove `#[allow(clippy::needless_pass_by_value)]`